a2/test.ml

```diff
@@ -140,19 +140,34 @@ let suite =
         assert_equal (List.length (Part1.perm [1;2;3;4;5;6;7;8;9;10])) 3628800
       );
 
-    "average 1" >:: (fun _ -> 
+    "average 1" >:: (fun _ ->
         skip_if true "skip";
         assert_bool "bad average" (cmp_float ~epsilon:0.01 377.85 (Query.average data1) )
       );
 
+    "average 2" >:: (fun _ ->
+        skip_if true "skip";
+        assert_bool "bad average" (cmp_float ~epsilon:0.01 374.32 (Query.average data2) )
+      ); 
+
+    "average 3" >:: (fun _ ->
+        skip_if true "skip";
+        assert_bool "bad average" (cmp_float ~epsilon:0.01 312.519489 (Query.average data3) )
+      );
+
     "average 4" >:: (fun _ -> 
         skip_if true "skip";
         assert_bool "bad average" (cmp_float ~epsilon:0.01 416.63 (Query.average data4) )
       );
 
+    "average 5" >:: (fun _ -> 
+        skip_if true "skip";
+        assert_bool "average of empty list" (Query.average [] = 0.)
```

Comment:
This could be just an assert_equal, right?

Reply:
Yeah -- I haven't really become fluent with the oUnit stuff, so just quick-copied from the others. I think the description string is also semantically malformed (i.e. should describe the meaning of failure, right?).

Reply:
Yeah that's right